src/components/MetricsPane/charts/TimeSeriesBenthicCover.jsx

```diff
@@ -0,0 +1,124 @@
+import { useContext } from 'react'
+import Plot from 'react-plotly.js'
+
+import { ChartSubtitle, ChartWrapper, TitlesWrapper } from './Charts.styles'
+import { FilterProjectsContext } from '../../../context/FilterProjectsContext'
+import { MetricCardH3 } from '../MetricsPane.styles'
+import dashboardOnlyTheme from '../../../styles/dashboardOnlyTheme'
+
+const chartTheme = dashboardOnlyTheme.plotlyChart
+const chartThemeLayout = chartTheme.layout
+const categories = Object.keys(chartTheme.timeseriesCharts.benthicCoverColorMap)
+
+export const TimeSeriesBenthicCover = () => {
+  const { filteredSurveys } = useContext(FilterProjectsContext)
+
+  const groupedBenthicCategoryCountByYear = filteredSurveys.reduce(
+    (accumulator, { sample_date, protocols }) => {
+      const year = new Date(sample_date).getFullYear()
+      const recordProtocols = protocols || {}
+      const benthicCategories = {}
+
+      categories.forEach((category) => {
+        const avgBenthicCoverValues = Object.keys(recordProtocols)
+          .map((protocol) => {
+            const categoryData = recordProtocols[protocol]?.percent_cover_benthic_category_avg
+            return categoryData?.[category] ?? null
+          })
+          .filter((val) => val !== null)
```

Comment:
What about `undefined` or empty strings

Reply:
`empty` and `undefined` would still be a null in this case.